solarwinds_apm/sampler.py

```diff
@@ -316,79 +317,31 @@ def create_xtraceoptions_response_value(
 
         return ";".join(response)
 
-    def create_new_trace_state(
```

Comment:
`calculate_trace_state` is so much simpler that I figured we could compress and get rid of its helper `create_new_trace_state`